src/Resource.cs

```diff
@@ -284,9 +279,7 @@ public Resource_recipe()
 			this.left = 1.0;
 		}
 
-		/// <summary>
-		/// add an input to the recipe
-		/// </summary>
+		// add an input to the recipe
```

Comment:
Why remove the summaries ? summaries are preferred over comments

Reply:
Adding back.